akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -79,7 +79,9 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
         streamId: Int,
         state:    StreamState,
         outlet:   SubSourceOutlet[StreamFrameEvent]
-      ) extends BufferedOutlet[StreamFrameEvent](outlet)
+      ) extends BufferedOutlet[StreamFrameEvent](outlet) {
+        override def onDownstreamFinish(): Unit = () // ignore substream cancellation, FIXME: is that correct?
```

Comment:
If the question is about how to prevent downstream cancellation bullbing then yes, correct.
If question about http semantics here then I'm not sure, we'll know soon enough